.env

```diff
@@ -0,0 +1,6 @@
+REACT_APP_FIREBASE_API_KEY = AIzaSyC1XgTfRAOYxJssERuJROxasTjg_u4k0Qk
+REACT_APP_FIREBASE_AUTH_DOMAIN = quiz-app-9a40e.firebaseapp.com
+REACT_APP_FIREBASE_PROJECT_ID = quiz-app-9a40e
+REACT_APP_FIREBASE_STORAGE_BUCKET  = quiz-app-9a40e.appspot.com
+REACT_APP_FIREBASE_MESSAGESENDER_ID = 398413713870
+REACT_APP_FIREBASE_APPID = 1:398413713870:web:9219085995e1d01dd8db9f
```

Comment:
you can use [Vercel's](https://vercel.com/docs/concepts/projects/environment-variables) environment variable for storing keys

Reply:
I have to look for that, thanks though